Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -989,7 +1025,8 @@ function Get-ServiceFabricApplicationActionOldSdk
         $getApplicationParams['ApplicationName'] = $ApplicationName
     }
 
-    $apps = Get-ServiceFabricApplication @getApplicationParams
+    $apps = Invoke-ActionWithDefaultRetries -Action { Get-ServiceFabricApplication @getApplicationParams } `
```

Comment:
Why can't we do this:
$apps = Get-ServiceFabricApplicationAction @getApplicationParams